projects/admin/src/app/acquisition/api/acq-account-api.service.ts

```diff
@@ -98,4 +98,29 @@ export class AcqAccountApiService {
     return this._http.get<any>(apiUrl, { params });
   }
 
+
+  /**
+   * Allow to sort accounts to render it correctly (corresponding to hierarchical tree structure)
+   * @param accounts - the accounts to sort.
+   * @return Account list sorted as a hierarchical tree.
+   */
+  orderAccountsAsTree(accounts: Array<AcqAccount>): Array<AcqAccount> {
+    /** Append an account and children accounts into the `accounts` list */
```

Comment:
cosmetic: `//` for one line

Reply:
This is a doctring for inner function ;-)